### ISSUE TITLE
Accessibility: Add Screenreader Friendly Controls for Mod Selection

### ISSUE THREAD

**Issue by fclorenzo** [Nexus-Mods/Vortex#23397]
When trying to select mods/plugins in the mods table, pressing CTRL+a doesn't work (tested with NVDA screenreader, both in browse and focus mode). Additionally, screenreader users are unable to select a subset of items (EG selecting 20 of the 50 installed mods).

My suggestions are as follows:

- Checkboxes close to each mod/plugin entry, so we can select the individual items we'd like (don't forget to label these checkboxes with the name of the item they refer to).
- A "select all" button (can be permanent or start to appear once we selected at least one item).
- Make the context menu (right click options menu) permanently expanded when there's at least one item selected. Alternatively, make the context menu expandable also by pressing shift+f10 (something that currently doesn't happen, only the raw mouse left click expands it). For screenreader users, shift+f10 is the usual way to expand these kind of context menus, since we avoid having to use a mouse.

For users who don't like this, I imagine these options could be toggled in settings.

If anyone needs further clarification, I'd be more than happy to help!
